src/main/java/org/jitsi/utils/dsi/DominantSpeakerIdentification.java

```diff
@@ -1074,21 +1074,26 @@ public synchronized long getLastLevelChangedTime()
          * @return a <tt>byte</tt> array which represents the (history of) audio
          * levels received or measured for this <tt>Speaker</tt>
          */
-        byte[] getLevels()
+        String getLevels()
         {
             // The levels of Speaker are internally maintained starting with the
             // last audio level received or measured for this Speaker and ending
             // with the first audio level received or measured for this Speaker.
             // Unfortunately, the method is expected to return levels in reverse
```

Comment:
Since this is now for human consumption, I don't think this is so much "this method is expected to" as much as "we want it to" - maybe update the comment?